<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>581</height>
    </rect>
   </property>
   <widget class="QLabel" name="lbl_background1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>1041</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
border-image: url(:/image/moutain.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>280</width>
      <height>390</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,100);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(255,255,255,210);</string>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Username">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>210</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0);
border: none;
border-bottom: 2px solid rgba(105, 118, 132, 255);
color: rgba(255, 255, 255, 230);
padding-bottom: 7px;</string>
    </property>
    <property name="placeholderText">
     <string>User Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Pwd">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>290</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0);
border: none;
border-bottom: 2px solid rgba(105, 118, 132, 255);
color: rgba(255, 255, 255, 230);
padding-bottom: 7px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Login">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>380</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop: 0 rgba(20,47, 78,219), stop: 1 rgba(85,98, 112,226));
color: rgba(255,255,255,210);
border-radius: 5px; 
}
QPushButton#pushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop: 0 rgba(40,67, 98,219), stop: 1 rgba(105,118, 132,226));
color: rgba(255,255,255,210);
border-radius: 5px; 
}
QPushButton#pushButton:pressed{
	padding-left: 5px;
	padding-top: 5px;
	background-color: rgba(105,118,132,200);
}</string>
    </property>
    <property name="text">
     <string>L o g  I n</string>
    </property>
   </widget>
   <widget class="QLabel" name="btn_Close">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>5</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res2.qrc"/>
 </resources>
 <connections/>
</ui>
